releasenotes/notes/update-split-2q-unitaries-with-swap-557a1252e3208257.yaml

```
@@ -0,0 +1,8 @@
+---
+features_transpiler:
+  - |
+    The :class:`.Split2QUnitaries` transpiler pass has been upgraded to
+    handle the case where the unitary in consideration can be written
+    as a SWAP gate and two 1-qubit gates. In this case, it splits the
+    unitary and also applies virtual swapping similar to what is done in
+    :class:`.ElidePermutations`.
```

I would probably mention the new argument here to document that it is being introduced. Something like:

```suggestion
    :class:`.ElidePermutations`. This functionality can be controlled with a new argument,
    ``split_swap``, on the constructor of :class`.Split2QUnitaries` which can be used to disable
    splitting swap equivalent gates.
```
```

---

Done in a9bc3e1ea